### DISCUSSION TITLE
The Scientific Study of Personal Wisdom - From Contemplative Traditions to Neuroscience

### DISCUSSION THREAD

**Discussion by msantolini** [life-itself/community#564] [Research]
Hello research people 🤗 

I wanted to share here a book which I am reading now and find quite interesting and relevant to some interests we have on **leveraging contemplative practices into fostering more personal/collective wisdom**. 

<img width="200" alt="image" src="https://github.com/life-itself/community/assets/956597/7ebdf69c-1d09-4d7e-ae01-0d4634545987">

[Ferrari and Weststrate - 2013 - The Scientific Study of Personal Wisdom](https://github.com/life-itself/community/files/11623279/Ferrari.and.Weststrate.-.2013.-.The.Scientific.Study.of.Personal.Wisdom.From.Cont.pdf)

It also features a chapter on **perspectival knowing and relevance realization** from Verveake and Ferrari, summarized by the editors as follows:

> Intelligence is necessary but not sufficient for wisdom because intelligence is not sufficient for rationality. Rationality recursively applies intelligence to the problem of using intelligence well. Likewise, wisdom recursively applies rationality to the problem of using rationality well. **Wisdom is rationality transcending itself rationally so as to greatly enhance one’s ability to realize relevance**. In particular, people who so engage their rationality should experience more active open-mindedness, insight, self-regulation, and perspectival knowing

I felt it is a good impression to what I feel is a "**metamodern science**", involving a recursive, transjective approach to "framing framing", which I detail a bit more in recent [slides](https://docs.google.com/presentation/d/16FWUnjyb6Vm-dMFaH02ZiI0zMvPolghB7jl4_OPfV3k/edit?usp=sharing) of a talk I gave to a research group working in complex systems, cybernetics, agency from [Center Leo Apostel](https://clea.research.vub.be/). 

Have a good read and comments welcome!

**Comment by baouroux** [comment]
Very interesting Marc, thanks for sharing. Framing - framing reminds me of the 'construct aware'  stage descriptions from developmental psychology (Piaget, Kegan, Loevinger etc). I wonder if they do indeed connect.
BTW I'm collaborating with Ubiquity University for a second filming project, connected to their Masters and PhD programmes in Wisdom studies https://www.ubiquityuniversity.org/courses/ma-in-wisdom-studies/
I'll go through your slides and would be great to see if there are some possible bridges to build their between initiatives.
